hibernate-rx-core/src/test/java/org/hibernate/rx/configuration/RxConnectionPoolTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.rx.configuration;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.hibernate.cfg.AvailableSettings;
+import org.hibernate.rx.cfg.AvailableRxSettings;
+import org.hibernate.rx.containers.PostgreSQLDatabase;
+import org.hibernate.rx.service.RxConnectionPoolProviderImpl;
+import org.hibernate.rx.service.initiator.RxConnectionPoolProvider;
+import org.junit.Test;
+
+import io.vertx.pgclient.PgPool;
+import io.vertx.sqlclient.Pool;
+
+public class RxConnectionPoolTest {
+
+	@Test
+	public void configureWithPool() {
+		String url = PostgreSQLDatabase.getJdbcUrl().substring( "jdbc:".length() );
+		Pool pgPool = PgPool.pool( url );
+		Map<String,Object> config = new HashMap<>();
+		config.put( AvailableRxSettings.VERTX_POOL, pgPool );
+		RxConnectionPoolProvider rxPool = new RxConnectionPoolProviderImpl( config );
+		rxPool.getConnection().close();
+	}
+	
+	@Test
+	public void configureWithJdbcUrl() {
+		String url = PostgreSQLDatabase.getJdbcUrl();
+		Map<String,Object> config = new HashMap<>();
+		config.put( AvailableSettings.URL, url );
+		RxConnectionPoolProvider rxPool = new RxConnectionPoolProviderImpl( config );
+		rxPool.getConnection().close();
```

Comment:
I'm not sure what you are testing here.
The url is a static string the we create specifically for tests.

Does it even fail if the credential or something is wrong with in the JDBC? I think nothing happens in getConnection() unless you run a query. And close() is a null method.

Reply:
This is indeed a pretty simple test. If this test fails then pretty much every other test would also be failing. But in case we ever switch to a different default way of configuring the tests, I thought it would be good to have this configuration path tested explicitly here.